pixel/head.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+  (function(window) {
+    var smarthintkey = "{{settings.smarthintkey}}";
+    if (smarthintkey) {
+      var script = document.createElement("script");
+      script.type = "text/javascript";
+      script.async = true;
+      script.src = "https://service.smarthint.co/Scripts/i/infracommerce.min.js";
+      var s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(script, s);
+      window.smarthintkey = smarthintkey;
```

Comment:
I think it's better to declare this variable before adding the script

https://www.smarthint.co/en/vtex-integration/